bindings/java/nz/mega/sdk/DelegateMegaLogger.java

```diff
@@ -33,6 +33,10 @@
         this.listener = listener;
     }
 
+    MegaLoggerInterface getLogger() {
```

Comment:
The function name could be `getUserListener` like the corresponding function in all other delegate listeners.